shake.js

```diff
@@ -23,12 +21,28 @@ function shake() {
   'AFFKPS',
   'HLNNRZ',
   'DEILRX',]
+
+function shuffle(array) {
+  for (let x = 0; x < 16; x++) {
+    let y = Math.floor(Math.random() * (x + 1));
+    [array[x], array[y]] = [array[y], array[x]];
+  }
+}
+//source for shuffle: https://javascript.info/task/shuffle
+
+function shake() {
+  shuffle(dice)
```

Comment:
Shoot this is kind of a random sanity check that shouldn't be included... Sorry!